ui/components/multichain/connected-site-permissions-modal/index.scss

```diff
@@ -0,0 +1,4 @@
+.connected-site-permissions-pill {
+  flex: 0 0 auto;
+  margin: auto;
```

Comment:
I think we can remove the scss here and use the utility Box Props to style this component. wdyt?